authorizer-app-backend/src/main/java/org/radarbase/authorizer/api/RestSourceUserMapper.kt

```diff
@@ -25,7 +25,7 @@ class RestSourceUserMapper {
         projectId = user.projectId,
         userId = user.userId,
         sourceId = user.sourceId,
-        isAuthorized = user.authorized,
+        isAuthorized = user.hasValidToken(),
```

Comment:
This is not the intention though. If `isAuthorized` is false, that indicates that the user needs to manually log in again. But if there still is a valid refresh token, no user interaction is needed.